sites/all/modules/ar/ar_assessments/js/ar_assessments.js

```diff
@@ -405,7 +405,7 @@ Drupal.behaviors.arAssessmentsAssessments = {
       },
 
       defaultRoute: function (actions) {
-        this.navigate('list/1', {trigger: true});
```

Comment:
@guillaumev yep, the discussion on HRINFO-594 came after that ticket. The table view is now a lot tidier so we're going to try it out with just that, but will be easy to revert if people prefer the list view.